### ISSUE TITLE
Can't register any Contour Next Link 2.4

### ISSUE THREAD

**Issue by KlausStoertebeker** [pazaan/600SeriesAndroidUploader#172]
I'm using 600SeriesAndroidUploader for the first time and started with version 0.5.1 from the beginning. Because there is NO description how to register my Contour Next Link 2.4 (there is a description in [https://nightscout.gitbooks.io/user_guide/content/de/grundlagen/640g/640gAndroidUploader.html](url). But this seems to be for an older version with a complete different user interface) I did "try and error".
After getting my Contour Next Link 2.4 recognized by 600SeriesAndroidUploader ("Contour Next Link plugged in."; by the way: I also got in touch with issue #156 "USB permission will not persist when requested from the app" :-( ) I selected the menu entry "Registered devices" to register my Contour Next Link 2.4. As expected, the list was empty ("No registered Contour Next Link devices").
And I don't know how to realize the information how to register my Contour Next Link 2.4: "To register a Contour Next Link you must first plug it in, **and get a reading from the pump.**")
My Contour Next Link 2.4 is plugged in and recognized by 600SeriesAndroidUploader, but how do I get a reading from the pump? If I select the menu entry "Read data now", I get the message "Requesting poll now..." but nothing happens.
So: How do I register my Contour Next Link 2.4?

#### Your Environment

* Uploader Version Number: 0.5.1
* Your Android Phone Model Name: Sony Xperia Z5 compact (E5823)
* Android Version Number: 6.0.1
* Network Connection at the time the issue occurred: not relevant

#### Brief Explanation of Issue
Can't register any Contour Next Link 2.4

#### Uploader Error Code
none

#### Steps to Reproduce (for bugs)
see explanation of issue

#### Severity Score
3

**Comment by Pogman** [comment]
Registering is not something you have to do specifically and you should just be able to plug the CNL in and get a reading. It will not show on the registered devices list until the first successful glucose value is retrieved from the pump.

Check the pairing between the CNL and the Pump, ie can you get a BG on the CNL and auto sent it to the pump.

**Comment by volkerrichert** [comment]
Maybe we need to update the Nightscout doc to match the current version of the uploader. Did anybody know how that can be done? Via PR, i think?

I can translate it into German, of course. But I't able to write a good documentation.

**Comment by volkerrichert** [comment]
Ah, ok. There is only a german version. :-(

**Comment by KlausStoertebeker** [comment]
@Pogman 
Pairing between CNL ans 640G is ok. I use the CNL at least three times a day for sensor calibration. How long should it take normally to get the first reading with an unregistered CNL?

**Comment by KlausStoertebeker** [comment]
Sorry - pressed the wrong button...

**Comment by Pogman** [comment]
@KlausStoertebeker it should be fast like within 30 seconds from first use as long as there is a SGV available on the pump. You can try clearing the app data and turning off/on (not restart) the phone. If your still having an issue try another OTG cable. Based on comments on FB this phone should work for you.

**Comment by Pogman** [comment]
@volkerrichert yeah docs are getting out of date and could do with work - at least to remove any of the confusing old references for setting up.

There are various fixes for lots of the minor things that can catch new users out in the next version.

**Comment by volkerrichert** [comment]
@Pogman : FACK

**Comment by pazaan** [comment]
Assigned to myself for documentation refresh.

**Comment by wright-david** [comment]
Responding specifically about the documentation angle:
I can't find anything at the URL given above (or https://legacy.gitbook.com/book/nightscout/user_guide/details), but I guess it was referring to an earlier version of the 600SeriesAndroidUploader wiki.

The original FAQ instructions about registering via Carelink were 'retired' to a sub-page in April 2017.
https://github.com/pazaan/600SeriesAndroidUploader/wiki/FAQ-for-releases-before-v0.4.0

**Comment by jothi1971** [comment]
I plan to use my android mobile to communicate with Medtronic 630G pump via CNL 2.4 BG meter.
Basically i will decide on how much bolus i need to set and send Remote bolus command. But i am uaable to find the payload parameters for this command. I only got command value as 0x0100. Please let me know the doc which specifies parameters and format

**Comment by Pogman** [comment]
@jothi1971 remote bolus commands can not be sent with comms through the usb interface to the CNL as they are blocked.

**Comment by jothi1971** [comment]
Hi
Thanks for the reply. Does this mean we can not do Looping using Medtronic
630G Pump and CNL?
Is there any other way we can Loop with Medtronic 630G pump?
Best Regards
jothi

**Comment by Pogman** [comment]
No know solution for using 600 Series pumps for looping.
